README.md

```diff
@@ -152,6 +155,40 @@ $ docker run -e CLASSPATH="/opt/hazelcast/CLASSPATH_EXT/*" -v /home/ubuntu/hazel
 
 Alternatively, you can [extend Hazelcast base image](#extending-hazelcast-base-image) adding your custom JARs.
 
+
+### Using TLS (Hazelcast Enterprise Only)
+
+The `HZ_NETWORK_SSL_ENABLED` environment variable can be used to enable TLS for the communication. You can check TSL related section at the documantation, [here](https://docs.hazelcast.org/docs/4.1.1/manual/html-single/index.html#integrating-openssl-boringssl).
```

Comment:
I guess what you describe here (and in point 1) is related to OpenSSL, which we actually don't recommend. I guess point 2 should be enough to use TLS with the standard Java TLS implementation.

Reply:
yes, you are totally right, done 👍